src/app/core/store/collections/collections.effects.ts

```diff
@@ -115,10 +114,10 @@ export class CollectionsEffects {
     withLatestFrom(this.store$.select(selectOrganizationByWorkspace)),
     map(([action, organization]) => {
       if (action.payload.error instanceof HttpErrorResponse && action.payload.error.status == 402) {
-        const title = this.i18n({id: 'serviceLimits.trial', value: 'Free Service'});
+        const title = this.i18n({id: 'serviceLimits.trial', value: 'Trial Service'});
```

Comment:
Are these changes intentional or a result of a bad merge?

Reply:
Bad merge :)